spec/controllers/reservations_controller_spec.rb

```diff
@@ -6,28 +6,27 @@
   render_views
 
   before(:all) do
-    @app_config = FactoryGirl.create(:app_config)
-
     @user = FactoryGirl.create(:user)
     @banned = FactoryGirl.create(:banned)
     @checkout_person = FactoryGirl.create(:checkout_person)
     @admin = FactoryGirl.create(:admin)
   end
 
   after(:all) do
-    User.delete_all
-    AppConfig.delete_all
-    Reservation.delete_all
-    Category.delete_all
-    EquipmentModel.delete_all
-    EquipmentItem.delete_all
+    User.destroy_all
+    Category.destroy_all
+    EquipmentModel.destroy_all
+    EquipmentItem.destroy_all
   end
 
   before(:each) do
+    @ac = mock_app_config(**FactoryGirl.attributes_for(:app_config),
```

Comment:
It's the second one -- we need a functional config to create valid users and reservations. In most of the specs I was able to hunt down the settings that were used and only include those attributes in the mock, but there were too many for the reservations controller so I just added everything from the factory.